corehq/apps/hqwebapp/static/hqwebapp/js/multiselect_utils.js

```diff
@@ -45,8 +45,17 @@ hqDefine('hqwebapp/js/multiselect_utils', function () {
         multiselectId,
         selectableHeaderTitle,
         selectedHeaderTitle,
-        searchItemTitle
+        searchItemTitle,
+        isOwner
     ) {
+        if (!isOwner) {
+            $('#' + multiselectId).hide().after(
+                $('#' + multiselectId).children().each(function (el) {
+                    $(el).text();
```

Comment:
Does this do anything? It looks like it just evaluates to some string but doesn't do anything with that.

Reply:
It does display it on the page as line-separated plain text

Reply:
Can you double-check that, and double-check that this code is actually what's doing that (by also commenting it out and seeing if it still works). I can't see how this would do that.